releases/point-2023-2-release-note.md

```diff
@@ -0,0 +1,86 @@
+---
+description: This article lists new features, improvements, and bug fixes in SysKit Point version 2023.2.
+--- 
+
+# SysKit Point 2023.2
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2023.2.0
+
+**Build number:** xxx
+
+**Release date:** May 11, 2023
+
+## Get the Latest Version
+
+### SysKit Point
+
+* **New to SysKit Point?** Get the latest SysKit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point).<br/>
+   Find all steps described in detail in the [deployment article](../set-up-point-data-center/deployment/deploy-syskit-point.md).
+    
+* **Already using SysKit Point?** Upgrade to the latest SysKit Point version. <br/>
+[![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FARMTemplates%2FPointUpdateDeploy%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../set-up-point-data-center/deployment/upgrade-syskit-point.md).
+
+## Features
+
+* **You can now resolve access and provisioning requests in the SysKit Point web app**! Whether using the SysKit Point Teams app or the SysKit Point web app, you can now resolve these tasks in any of the two places. 
+As before, you will:
+    * **Get a notification in the Teams app**
+    * **Receive an email that opens the request in SysKit Point Teams app** 
+    * **Be able to resolve access and provisioning requests within SysKit Point Teams app**
+In the newest addition, you will:
+    * **Get a task in the SysKit Point app as well**
+    * **Be able to navigate to a task in the SysKit Point web app from the received email**
+    * **Be able to resolve access and provisioning requests within the SysKit Point web app**
+
+* **Group email address and site address editing added to provisioning requests**!
+    * **When requesting a Microsoft Team or a Microsoft 365 Group**, you can **edit the group email address**. 
+    * **When requesting a new site**, you can now **edit the site address**, thus affecting the site's URL.
```

Comment:
site URL?

Reply:
instead of site address I mean

Reply:
Wanted to offer both terms in case someone doesn't get it if I only talk about site address.